.travis.yml

```diff
@@ -4,6 +4,13 @@ language: minimal
 
 services: docker
 
+branches:
+    only:
+        - master
```

Comment:
better way to instrument Travis CI than the point and click interface, allows the mainline and PRs to be built but not all pushed branches